tests/State/ScopeTest.php

```diff
@@ -237,7 +282,8 @@ public function testClear(): void
         $scope->setFingerprint(['foo']);
         $scope->setExtras(['foo' => 'bar']);
         $scope->setTags(['bar' => 'foo']);
-        $scope->setUser(['foobar' => 'barfoo']);
+        $merge = true;
```

Comment:
Instead of using a variabile just pass `true` as value of the 2nd argument of the `setUser` function

Reply:
04406b5